### DISCUSSION TITLE
Polyline has wrong position after being serialized and deserialized

### DISCUSSION THREAD

**Discussion by MethDamon** [fabricjs/fabric.js#7097] [Q&A]
Hi, I'm trying to add polylines to my canvas via dynamically adding points to the canvas with mouse clicks.

I store the first point as "left" and "top" coordinates and push {x: 0, y: 0} to the polyline's points.

This works perfectly fine if the first point is also the top most and left most point of all points, but in all other cases the polyline jumps around the canvas after being serialized and deserialized again.

If someone could help me point me in the right direction on how to handle this problem I'd be very thankful.

**Comment by asturur** [comment]
Hi @MethDamon please replicate in a jsfiddle in the simplest way possible what are you doing if you want someone to have a look at it.

I do not see serialization involved here anyway.
